crates/ruff_python_semantic/src/cfg/mod.rs

```diff
@@ -0,0 +1,61 @@
+pub mod graph;
+pub mod visualize;
+
+#[cfg(test)]
+mod tests {
+    use std::fmt::Write;
+    use std::fs;
+    use std::path::PathBuf;
+
+    use crate::cfg::graph::build_cfg;
+    use crate::cfg::visualize::draw_cfg;
+    use insta;
+    use ruff_python_ast::Stmt;
+    use ruff_python_parser::parse_module;
+    use ruff_text_size::Ranged;
+    use test_case::test_case;
+
+    #[test_case("no_flow.py")]
+    fn control_flow_graph(filename: &str) {
+        let path = PathBuf::from_iter(["resources/test/fixtures/cfg", filename]);
```

Comment:
Is this the same as?

```suggestion
        let path = PathBuf::from("resources/test/fixtures/cfg").join(filename);
```